src/main/java/Calendar1/Main.java

```diff
@@ -0,0 +1,108 @@
+package Calendar1;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] input = new int[]{
```

Comment:
What is the input for?

Reply:
Array with input number from task. Why it's wrong?

Reply:
In the context of a rocket, it has a meaning better than semantic input